webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -226,15 +220,12 @@ export const GlideTable: React.FC<GlideTableProps> = ({
 
   useEffect(() => {
     const selectedRowIndices = selection.rows.toArray();
-    setSelectedExperimentIds((prevIds) => {
-      const selectedIds = selectedRowIndices
-        .map((idx) => data?.[idx])
-        .filter((row) => row !== undefined)
-        .filter(Loadable.isLoaded)
-        .map((record) => record.data.experiment.id);
-      if (prevIds === selectedIds) return prevIds;
-      return selectedIds;
-    });
+    const selectedIds = selectedRowIndices
```

Comment:
Why did you change this?

Reply:
The function `setSelectedExperimentIds` which was passed down from the `useState` in `F_ExperimentList`, got replaced with a new function `handleExperimentSelection` where we also call `updateSettings`. Because we call it with `setSelectedExperimentIds([])` I want this to consistently receive an array (not `(prev) => response` form)

Reply:
It did consistently receive an array before. Actually, the `if (prevIds === selectedIds) return prevIds;` never fired because it was comparing object ids so I guess this code is equivalent.